index.yaml

```diff
@@ -2849,24 +2785,6 @@ entries:
     - assets/rancher-node-exporter/rancher-node-exporter-100.0.0+up1.16.2.tgz
     version: 100.0.0+up1.16.2
   rancher-operator:
-  - annotations:
-      catalog.cattle.io/auto-install: rancher-operator-crd=match
-      catalog.cattle.io/certified: rancher
-      catalog.cattle.io/experimental: "true"
-      catalog.cattle.io/hidden: "true"
-      catalog.cattle.io/namespace: rancher-operator-system
-      catalog.cattle.io/os: linux
-      catalog.cattle.io/provides-gvr: clusters.rancher.cattle.io/v1
-      catalog.cattle.io/release-name: rancher-operator
-    apiVersion: v2
-    appVersion: 0.1.4
-    created: "2021-06-16T21:05:17.448049-07:00"
-    description: Control Rancher using GitOps
-    digest: e7400d81d182a88cd50cd8b076191a4b1eca7ff1004865795349fdaa2a705e53
-    name: rancher-operator
-    urls:
-    - assets/rancher-operator/rancher-operator-100.0.0+up0.1.4.tgz
-    version: 100.0.0+up0.1.4
```

Comment:
100.0.0+up0.1.6 is deleted because the package dir of rancher-opertor is deleted. 
the rest are pulled from release-v2.6 branch.